readme.md

```diff
@@ -0,0 +1,32 @@
+# Bond Fight - Node
+
+### Setup
+
+1. Clone Node repository by running `git clone git@github.com:Mayden-Academy/2019-nodeApi-bondFight.git`
+2. Create MongoDb database with name Films and collection Bond_Films, and populate with `bond.json` from db folder
+3. Turn on mongodb by running `mongod --config /usr/local/etc/mongod.conf`
+4. On the command line, cd into node directory and run `node index.js`
+
+
+### Routes
+
+/Bond_Films
+GET
+
+- Gets all films from database to hydrate card components within card container. 
```

Comment:
not always gonna be passed to card container - go more generic - Gets all films and returns array of objects